client/jetpack-cloud/sections/manage/pricing/license-products-list/featured-license-multi-item-card.tsx

```diff
@@ -0,0 +1,114 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { Button } from '@automattic/components';
+import 'calypso/my-sites/plans/jetpack-plans/product-store/featured-item-card/style.scss';
+import { useTranslate } from 'i18n-calypso';
+import { useCallback, useState } from 'react';
+import MultipleChoiceQuestion from 'calypso/components/multiple-choice-question';
+import { useProductDescription } from 'calypso/jetpack-cloud/sections/partner-portal/hooks';
+import getProductShortTitle from 'calypso/jetpack-cloud/sections/partner-portal/lib/get-product-short-title';
+import getProductVariantShortTitle from 'calypso/jetpack-cloud/sections/partner-portal/lib/get-product-variant-short-title';
+import { HeroImageAPIFamily } from 'calypso/my-sites/plans/jetpack-plans/product-store/hero-image';
+import { useDispatch } from 'calypso/state';
+import { APIProductFamilyProduct } from 'calypso/state/partner-portal/types';
+import { ItemPrice } from './item-price';
+import './style.scss';
+
+type FeaturedLicenseMultiItemCardProps = {
+	items: APIProductFamilyProduct[];
+	bundleSize?: number;
+	ctaAsPrimary?: boolean;
+	ctaHref?: string;
+	moreInfoLink?: React.ReactNode;
+	isCondensedVersion?: boolean;
+	isCtaDisabled?: boolean;
+	isCtaExternal?: boolean;
+	onClickCta?: VoidFunction;
+};
+
+export const FeaturedLicenseMultiItemCard = ( {
+	items,
+	bundleSize,
+	ctaAsPrimary,
+	ctaHref,
+	moreInfoLink,
+	isCtaDisabled,
+	isCtaExternal,
+	onClickCta,
+}: FeaturedLicenseMultiItemCardProps ) => {
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+
+	const [ item, setItem ] = useState( items[ 0 ] );
+
+	let title = item.name;
+	if ( title.startsWith( 'Jetpack Security' ) ) {
+		title.replace( 'Jetpack Security', 'Security' );
+	}
+	if ( title.startsWith( 'Jetpack VaultPress' ) ) {
+		title.replace( 'Jetpack VaultPress', 'VaultPress Backup' );
+	} else {
+		title = getProductShortTitle( item, true );
+	}
+	const ctaLabel = translate( 'Get' );
+	const ctaAriaLabel = ctaLabel + ' ' + item.name;
+
+	const price = <ItemPrice bundleSize={ bundleSize } item={ item } />;
+	const { description: productDescription } = useProductDescription( item.slug );
+	const hero = <HeroImageAPIFamily item={ item } />;
+
+	const onChangeOption = useCallback(
+		( selectedProductSlug: string ) => {
+			const selectedProduct =
+				items.find( ( { slug } ) => slug === selectedProductSlug ) ?? items[ 0 ];
+
+			setItem( selectedProduct );
+			dispatch(
+				recordTracksEvent( 'calypso_jp_manage_pricing_page_variant_option_click', {
+					product: item.slug,
+				} )
+			);
+		},
+		[ dispatch, item.slug, items ]
```

Comment:
No need to include this as a dependency or use it in Tracks, as it's the same as `selectedProduct.slug` and `selectedProductSlug`, right?

Reply:
Good point! Fixed in e9dd3afd5dbbda87b34d96ab04efea54e203f064.